assets/jsx/hasher.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import filesize from 'filesize'
+
+
+class Hasher extends React.Component {
```

Comment:
Yeah, it just stood out to me during the review and I figured that didn't bode well for anyone less familiar with the project :grinning: